docs/registry/create/_index.md

```diff
@@ -300,15 +301,24 @@ class MyBase(Base, Reconfigurable):
             self.right.is_powered(extra=extra, timeout=timeout)[0]
 ```
 
+</details>
+<br>
+
+Additional example modules are available in the [Python SDK GitHub repository](https://github.com/viamrobotics/viam-python-sdk/tree/main/examples).
+
 {{% /tab %}}
 {{% tab name="Go"%}}
 
-The Go code for the custom model (<file>mybase.go</file>) and module entry point file (<file>main.go</file>) is adapted from the full demo modules available on the [Viam GitHub](https://github.com/viamrobotics/rdk/blob/main/examples/customresources).
+Your new resource model server must have all the methods that the Viam RDK requires, and should match the built-in API client {{< glossary_tooltip term_id="subtype" text="subtype" >}} like ['rdk:component:base'](https://pkg.go.dev/go.viam.com/rdk/components/base#pkg-functions).
+
+Create a folder for your module and save your code as a file named <file>my_modular_resource.go</file> inside.
+
+The following example module registers a modular resource implementing Viam's built-in [Base API](/components/base/#api) [(rdk:service:base)](/registry/key-concepts/#models) as a new model, `"mybase"`, using the model family `acme:demo:mybase`.
 
 <file>mybase.go</file> implements "mybase", a custom model of the base component, and registers the new model and API helper functions with the Go SDK.
```

Comment:
Opt: could remove this line as well in favor of line 316, though there's a but more usefulness to go's version here.